pkg/autoupdate/tuf/autoupdate.go

```diff
@@ -0,0 +1,236 @@
+package tuf
+
+// This new autoupdater points to our new TUF infrastructure, and will eventually supersede
+// the legacy `Updater` in pkg/autoupdate that points to Notary.
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/version"
+	client "github.com/theupdateframework/go-tuf/client"
+	filejsonstore "github.com/theupdateframework/go-tuf/client/filejsonstore"
+)
+
+//go:embed assets/tuf-dev/root.json
+var rootJson []byte
+
+const (
+	DefaultTufServer       = "https://tuf-devel.kolide.com"
+	DefaultChannel         = "stable"
```

Comment:
Exporting these seems weird.

Reply:
Looks like `DefaultChannel` was exported from a previous iteration -- unexported it. (`DefaultTufServer` is in use in `options`, matches similar consts for notary/mirror URLs that live in `autoupdate`.)